### DISCUSSION TITLE
Prevent error messages by piping to xargs

### DISCUSSION THREAD

**Discussion by jebarpg** [typicode/husky#1375] [Ideas]
on this page
https://typicode.github.io/husky/how-to.html
Modify the prepare script to never fail:

json
// package.json
"prepare": "husky || true"
You'll still get a command not found error message in your output which may be confusing.

up can prevent the error message from showing by piping to xargs

like so

"prepare": "husky || true | xargs"

**Comment by typicode** [comment]
Nice!

It probably won't work on Windows unfortunately :/
